### PR TITLE
Add entry logger scanner stats

### PR DESCRIPTION
**Pull request by wuzhanpeng**
Descriptions of the changes in this PR:

### Motivation

In our practice, when the throughput becomes larger, the trigger frequency of compaction will also increase, so that the scanning of entry log will also become frequent. It is well known that frequent read operations in HDD are a common cause of disk performance degradation, but we currently have no relevant monitoring information on this. Therefore, this pr provides a convenient way to provide additional monitoring of log scanning.

### Changes

Add two counters for log scanning monitoring.

### CODE REVIEW

**Comment by wuzhanpeng** [comment]
rerun failure checks

**Comment by wuzhanpeng** [comment]
rerun failure checks

**Comment by wuzhanpeng** [comment]
rerun failure checks

**Comment by wuzhanpeng** [comment]
rerun failure checks

**Comment by wuzhanpeng** [comment]
@zymap  Would you mind to help review it?

**Comment by hangc0276** [comment]
@wuzhanpeng  Thanks for your contribution, would you please rebase the master? thanks.

File: bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/EntryLogger.java [hunk 1]
```diff
@@ -1038,6 +1078,7 @@ public void scanEntryLog(long entryLogId, EntryLogScanner scanner) throws IOExce
                 }
                 data.capacity(entrySize);
                 int rc = readFromLogChannel(entryLogId, bc, data, pos);
+                entryLogCounter.numOfScannedBytes.add(data.writableBytes());
```
Comment (hangc0276) [inline bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/EntryLogger.java:1081]: data.readableBytes()?

**Review by hangc0276 — COMMENTED** [review]
The scannedByte is not the real bytes read from disks because it uses BufferedReadChannel to do prefetch, and the OS PageCache also has prefetch. Please refer to https://github.com/apache/bookkeeper/pull/3192

**Comment by zymap** [comment]
@wuzhanpeng Do you want to continue to work on this? If not, I will remove the release label

**Comment by zymap** [comment]
Move it to the next release since this PR is inactive.

**Comment by StevenLuMT** [comment]
fix old workflow,please see #3455 for detail

**Comment by shoothzj** [comment]
closed by inactive, feel free to reopen it if you still want to working on this :)

**Pull request closed without merging**